packages/terra-dev-site/src/layouts/secondary-navigation-layout/side-nav/SideNavHeader.module.scss

```diff
@@ -0,0 +1,47 @@
+@import '~terra-mixins/lib/Mixins';
+@import './clinical-lowlight-theme/SideNavHeader.module';
+@import './orion-fusion-theme/SideNavHeader.module';
+
+:local {
+  .page-header-container {
+    box-shadow: var(--terra-dev-site-side-nav-header-page-header-container-box-shadow, inset -3px -4px 0 -3px #fefffe, inset -3px -5px 0 -3px #c7cacb);
```

Comment:
there have been a couple of lint fixes applied, worryingly style lint wasn't running correctly in the single repo.